data/test/scenarios/test_move_unit.cfg

```diff
@@ -280,6 +298,8 @@
             dir=sw,s,se,ne,n,nw
         [/move_unit]
 
+# The assert can't check that Alice moved, but it does check that the implementation of
+# [move_unit] didn't choose an unoccupied hex next to the (initially-occupied) starting hex.
```

Comment:
Please run wmlindent on this file (it isn't run automatically on the test dir, IIRC because it doesn't handle some of the Lua code correctly).